README.md

```diff
@@ -203,6 +203,7 @@ elif ((current_hour >= 17 && current_hour < 24)) || ((current_hour >= 0 && curre
 fi
 ```
 
+<<<<<<< HEAD
```

Comment:
This appears to be an artifact of the merge conflict, can we remove this? I'm assuming we don't want to add it to the real file.